CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -153,6 +179,11 @@ local function OnCharacterAdded(character)
 	
 	backpack.ChildAdded:connect(OnChildAdded)
 	backpack.ChildRemoved:connect(OnChildRemoved)
+
+	if humanoid then
+		humanoid.Changed:connect(function() UpdateHealth(humanoid) end)
+		UpdateHealth(humanoid)
+	end
```

Comment:
To be truly bulletproof we should account for the potential addition or removal of Humanoid from the character.